src/components/menu/sub-menu.tsx

```diff
@@ -97,26 +106,50 @@ function SubMenu({
   // On mobile, the sub-menu should stay open if a user is on one of the links in that menu
   let showItemsClass = showItems || childIsCurrent() ? 'show' : '';
   return (
-    <li
-      tabIndex={0}
-      onKeyDown={e => (SUPPORTED_KEYS.includes(e.key)) && setActive(!active)}
-      onClick={toggleShowItems}
-      onMouseEnter={handleShowSubMenu(true)}
-      onMouseLeave={handleShowSubMenu(false)}
-      onTouchStart={handleShowSubMenu(!showSubMenu)}
-      className={`menu-item ${isCurrent} ${className} ${active || showSubMenu ? 'active' : ''} ${showItemsClass}`}
-    >
-      <span>
-        {label}
-        <ChevronLeft className="submenu-chevron" />
-      </span>
-      <Menu isSubMenu>
-        {/* Need to pass through toggleMenu so that the menu closes when a MenuLink is clicked */}
-        {Children.map(children, child => (
-          cloneElement(child as ReactElement, { toggleMenu })
-        ))}
-      </Menu>
-    </li>
+    <>
+      {/* desktop submenu */}
+      <li
+        tabIndex={0}
+        onKeyDown={e => (SUPPORTED_KEYS.includes(e.key)) && setActive(!active)}
+        onClick={toggleShowItems}
+        onMouseEnter={handleShowSubMenu(true)}
+        onMouseLeave={handleShowSubMenu(false)}
+        onTouchStart={handleShowSubMenu(!showSubMenu)}
+        className={`desktop-menu menu-item ${isCurrent} ${className} ${active || showSubMenu ? 'active' : ''} ${showItemsClass}`}
+      >
+        <span>
+          {label}
+          <ChevronLeft className="submenu-chevron" />
+        </span>
+        <Menu isSubMenu>
+          {/* Need to pass through toggleMenu so that the menu closes when a MenuLink is clicked */}
+          {Children.map(children, child => (
+            cloneElement(child as ReactElement, { toggleMenu })
+          ))}
+        </Menu>
+      </li>
+
+      {/* mobile submenu */}
```

Comment:
issue 5 in ticket fix. Mobile submenu doesn't have the onMouseEnter and onMouseLeave to remove hovering functionality.